.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ "*" ]
+    paths-ignore: [ "docs/**" ]
+
+  push:
+    branches: [ "*" ]
+    paths-ignore: [ "docs/**" ]
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  pytest:
+    strategy: 
+      matrix:
+        python-version: ['3.8', '3.6']
+
+    runs-on: ubuntu-20.04
+    steps:
+
+      - name: Checkout Code Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{matrix.version}}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      
+      - name: Run coverage
+        run: |
+          coverage run --source="." -m pytest
+          coverage report --fail-under=72 -m
+
+      - name: Run quality analysis
+        run: |
+          pylint --exit-zero seb_openedx
```

Comment:
I know noticed this --exit-zero flag.

Is there a reason for this? shouldn't we run pylint normally and use exceptions where the errors are?

Reply:
This flag allows github actions to know pylint has finished gracefully, we can also use pylint-exit:

https://stackoverflow.com/questions/71227177/github-action-error-process-completed-with-exit-code-14

Reply:
@felipemontoya ready